packages/benchmarks/src/quizQuestions/bin/reportMongoDbUniversityBadgeQuestionBenchmark.ts

```diff
@@ -0,0 +1,230 @@
+import { BRAINTRUST_ENV_VARS } from "../../envVars";
+import { assertEnvVars } from "mongodb-rag-core";
+import {
+  ExperimentResult,
+  getBraintrustExperimentResults,
+} from "../../reporting/getBraintrustExperimentResults";
+import path from "path";
+import fs from "fs";
+import { createObjectCsvWriter } from "csv-writer";
+import { aggregateScoresByTag } from "../../reporting/aggregateScoresByTag";
+import { convertTagStatsToFlatObject } from "../../reporting/convertTagStatsToFlatObject";
+import {
+  QuizQuestionEvalCase,
+  QuizQuestionTaskOutput,
+} from "../QuizQuestionEval";
+import { aggregateExperimentScoreMean } from "../../reporting/aggregateExperimentScoreMean";
+
+const { BRAINTRUST_API_KEY } = assertEnvVars(BRAINTRUST_ENV_VARS);
+const projectName = "mongodb-multiple-choice";
+const experiments = [
+  {
+    model: "GPT-4o",
+    experimentName: "gpt-4o-badge-631d3a9b",
```

Comment:
Where are these hash values coming from? e.g. `631d3a9b`

Reply:
that's braintrust generated. for if you have multiple experiments w/ the same name